src/main/java/models/Game.java

```diff
@@ -60,30 +68,73 @@ public void shuffle()
         Collections.shuffle(deck, new Random(seed));
     }
 
-    public abstract void initialDeal();
 
     public int getTotalScore(ArrayList<String> scores)
     {
         int total=0;
+        int totalAces=0;
 
         for(int i=0;i<scores.size();i++)
         {
             if (scores.get(i).equals("A"))
             {
-                total+=11;
-                if (total>21)
-                {
-                    total-=11;
-                    total+=1;
-                }
+                totalAces+=1;
             }
             else
             {
                 total+=Integer.valueOf(scores.get(i));
             }
         }
+
+        for(int i=0;i<totalAces;i++)
+        {
+            if (total>21)
+            {
+                total+=1;
+                continue;
+            }
+            total+=11;
+            if (total>21)
+            {
+                total-=11;
+                total+=1;
+            }
+        }
         return total;
     }
 
+    public String checkWinner(Game dealer,Game player)
+    {
+        if (player.playerScore > player.splitScore)
+        {
```

Comment:
When we split, we have two columns. So, we need to make sure that we're taking the highest score in the split columns to compare with the dealer.